src/wallet/rpcwallet.cpp

```diff
@@ -1126,8 +1126,6 @@ UniValue sendmany(const UniValue &params, bool fHelp)
                                        "\"{\\\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XZ\\\":0.01,"
                                        "\\\"1353tsE8YMTA4EuV7dgUXGjNFf9KpVvKHz\\\":0.02}\", 6, \"testing\""));
 
-    LOCK2(cs_main, pwalletMain->cs_wallet);
-
```

Comment:
I think these locks are needed.

Reply:
(as discussed in keybase) the locks are taken inside the functions